crash_similarity.py

```diff
@@ -76,11 +83,13 @@ def get_stack_trace_for_uuid(uuid):
     return data['proto_signature']
 
 
-def train_model(corpus):
-    if os.path.exists('stack_traces_model.pickle') and \
-       os.path.exists('stack_traces_model.pickle.docvecs.doctag_syn0.npy'):
-        return gensim.models.Doc2Vec.load('stack_traces_model.pickle')
+def train_model(corpus, embedding_algo='doc2vec'):
+    if os.path.exists('stack_traces_' + embedding_algo + '_model.pickle') and embedding_algo == 'doc2vec'\
```

Comment:
Since `os.path.exists('stack_traces_' + embedding_algo + '_model.pickle')` is in common between this and the following if, can you do something like:
```Python
if os.path.exists('stack_traces_' + embedding_algo + '_model.pickle'):
    if embedding_algo == 'doc2vec' ...:
        ...
    elif embedding_algo == 'word2vec':
        ...
```